### ISSUE TITLE
False positive in detecting PHP function

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#1274]
This is false positive:
```
wp_function_not_compatible_with_requires_wp: Function "serialize()" requires WordPress 6.7.0, but your plugin minimum supported version is WordPress 6.0.0.
```

Function "serialize" is actually PHP function and it is available since PHP 4.

Example code which triggered it:
```php
$wpdb->update(
    $wpdb->options,
    [ 'option_value' => serialize( $items ) ],
    [ 'option_name' => 'cron' ]
);
```

**Comment by ernilambar** [comment]
Looks like this is bug in generation wp-since JSON file.

In original repo:
```js
 "method:WP_HTML_Processor::serialize": {
        "type": "method",
        "since": "6.7.0",
        "file": "wp-includes/html-api/class-wp-html-processor.php"
    },
```

But here we have marked "serialize" as global functions.

https://github.com/WordPress/plugin-check/blob/trunk/includes/Vars/wp-functions-since.json#L2810
